web/src/components/snapshots/SnapshotSettings.jsx

```diff
@@ -116,15 +117,15 @@ class SnapshotSettings extends Component {
 
 
       if (this.state.checkForVeleroAndRestic) {
-        if (this.state.snapshotSettings?.isVeleroRunning !== lastState.snapshotSettings?.isVeleroRunning) {
-          this.setState({ veleroVersionUpdated: this.state.snapshotSettings?.isVeleroRunning });
+        if (this.state.snapshotSettings?.veleroPod !== lastState.snapshotSettings?.veleroPod && !isEmpty(this.state.snapshotSettings?.veleroPod)) {
```

Comment:
I don't think this is right.  The value should be compare to the state before the Save button was clicked, not to the state since the last check.

Reply:
It is like that...flag on line 119 becomes true when save button is clicked.

Reply:
cool